### PR TITLE
[#76] SOD offset now saved in DB

### PR DESCRIPTION
**Pull request by chimano**
- sod_offset added in messages collection
- last_sod_offset added in orderbooks collection

This is needed to add uniqueness to messages

### CODE REVIEW

File: core/scripts/models/order_book.py [hunk 1]
```diff
@@ -93,9 +96,10 @@ def _do_execute(self, msg: Message):
         else:
             self._do_delete(msg)
 
-    def send(self, msg: Message):
+    def send(self, msg: Message, last_sod_offset: int = 0):
```
Comment (Lercerss) [inline core/scripts/models/order_book.py]: Here the sod_offset could simply be extracted from the message passed as parameter

Comment (chimano) [reply core/scripts/models/order_book.py]: oh right the drowsiness got to me

Comment (chimano) [reply core/scripts/models/order_book.py]: Oh wait no. I did it like this so that the tests would not be affected by it

Comment (Lercerss) [reply core/scripts/models/order_book.py]: How about adding the default value to the message model instead?

Comment (chimano) [reply core/scripts/models/order_book.py]: Smart man

**Pull request merged**